test/integration/gardenlet/shootstate/secret/secret_test.go

```diff
@@ -42,10 +44,37 @@ var _ = Describe("ShootSecret controller tests", func() {
 		shootState *gardencorev1alpha1.ShootState
 
 		cluster *extensionsv1alpha1.Cluster
+
+		seedNamespace *corev1.Namespace
 	)
 
 	BeforeEach(func() {
-		resourceName = "test-" + utils.ComputeSHA256Hex([]byte(CurrentSpecReport().LeafNodeLocation.String()))[:8]
+		resourceName = "test-" + utils.ComputeSHA256Hex([]byte(uuid.NewUUID()))[:8]
+
+		By("creating seed namespace")
+		// name doesn't follow the usual naming scheme (technical ID), but this doesn't matter for this test, as
+		// long as the cluster has the same name
```

Comment:
Is this comment relevant here?

Reply:
Yes. The cluster name is usually the shoot technical ID which is not the case here.